moto/wafv2/responses.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
+
+import json
+
+from moto.core.responses import BaseResponse
+from .models import wafv2_backends, GLOBAL_REGION
+
+
+class WAFV2Response(BaseResponse):
+    SERVICE_NAME = "wafv2"
+
+    @property
+    def wafv2_backend(self):
+        return wafv2_backends[self.region]
+
+
+class WafV2Handler(BaseResponse):
+    def list_web_ac_ls(self):
+        wacl = wafv2_backends[GLOBAL_REGION]
```

Comment:
`self.body` returns `{"Scope": "CLOUDFRONT"}` or  `{"Scope": "REGIONAL"}`, so we can use that to determine which backend to talk to.

Reply:
Used `self.body` and added/updated tests.